src/main/java/com/tradeshift/flagr/Flagr.java

```diff
@@ -82,4 +89,51 @@ public EvaluationResponse evaluate(EvaluationContext context) {
         }
         return evalResponse;
     }
+
+    /*
+     * evaluate the flag and always return a Boolean value.
+     * <p>
+     * This method is useful if an on/off switch is needed.
+     * Be sure that the variant keys for the flag maps to
+     * the following values: true, false, enabled, disabled.
+     * The method is case insensitive and it will return a Boolean
+     * even when the variant key value is "tRue".
+     * */
+    public Boolean evaluateEnabled(EvaluationContext context) {
```

Comment:
Why is this not an Optional?

Reply:
@sorenmat Since it's always returning true or false, even on error, I don't see the need for it.